types/react-native/index.d.ts

```diff
@@ -5614,7 +5614,7 @@ export interface PixelRatioStatic {
  */
 export type PlatformOSType = 'ios' | 'android' | 'windows' | 'web'
 
-interface PlatformStatic {
+interface PlatformStatic extends PlatformIOSStatic {
```

Comment:
Shouldn’t this be the other way around and require the user to cast the `Platform` module to the iOS one if `OS = 'ios'` ?

Reply:
I'm actually not sure how to manage this piece with TS as Platform is for both but those attribute are only available for iOS. 

Should we have something like this?

```
export var Platform: PlatformStatic | PlatformIOSStatic;
```

Reply:
I was thinking something like the following:

```ts
interface PlatformIOSStatic extends PlatformStatic { ... }
```

```ts
import * as ReactNative from "react-native"

const Platform: PlatformIOSStatic = ReactNative.Platform
```

But I have no idea which would be more usable. Maybe give both options a try and feel which works best?